build/types/knockout.d.ts

```diff
@@ -329,8 +329,19 @@ export const extenders: Extenders<any>;
 
 //#region subscribables/mappingHelpers.js
 
-type Unwrapped<T> = T extends ko.Subscribable<infer R> ? R :
-    T extends Record<any, any> ? { [P in keyof T]: Unwrapped<T[P]> } : T;
+type Unwrapped<T> = T extends ko.ObservableArray<infer R>
```

Comment:
Rather than treating ObservableArray differently, this should be the pattern for any observable that contains an array.

Reply:
this first condition can actually be removed. `Record<any, any>` also matches arrays (they match the interface `Record<number, any>`).

Reply:
Looks like that's something new in 3.7.